src/Servers/Kestrel/test/InMemory.FunctionalTests/HttpsConnectionMiddlewareTests.cs

```diff
@@ -362,12 +362,13 @@ await using (var server = new TestServer(context => context.Response.WriteAsync(
         }
 
         [Fact]
-        public async Task DoesNotSupportTls10()
+        public async Task Tls10CanBeDisabled()
         {
             void ConfigureListenOptions(ListenOptions listenOptions)
             {
                 listenOptions.UseHttps(options =>
                 {
+                    options.SslProtocols = SslProtocols.Tls12 | SslProtocols.Tls12;
```

Comment:
```suggestion
                    options.SslProtocols = SslProtocols.Tls11 | SslProtocols.Tls12;
```

Reply:
I though we had decided to drop selection, and default to the OS like we advise everyone else to do?

Reply:
Right, but the way this is written now, this is just `SslProtocols.Tls12`. The same value was on the left *and* right side of the `|` operator, which looks like a typo, isn't it?